src/create-extension-pack.js

```diff
@@ -160,6 +162,19 @@ const externalBuiltins = ['ms-vscode.references-view',
     }
 })();
 
+/**
+ * @param {string} folderPath 
+ */
+function createYarnLock(folderPath) {
+    exec('yarn install', { cwd: folderPath }, (error, stdout) => {
```

Comment:
I think this is causing the `vsce` issue I have seen (see comment at call spot above). Here `exec()` returns a child process. I think we would need to wait for it to exit before returning, else it might not be done creating `yarn.lock`?

Reply:
I think you could use `paths:run()`, from this repo, instead. That one waits on the child process.

Reply:
I have just pushed a change so we use `paths:run()` as you have suggested and we `await` for it to return.
Good catch ! Thanks !!